0.3/schemas/json_schema/omero.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$id": "http://localhost:8000/omero.schema.json",
```

Comment:
Why is this not `0.3/schemas/json_schema/omero.schema` to match the previous naming scheme?

Reply:
This isn't actually used for validation any more, so I'll remove it.